From 587644a8567e6a9533f88401daa59cbd78c4632f Mon Sep 17 00:00:00 2001
From: "Justin T. Gibbs" <justing@spectralogic.com>
Date: Sat, 22 Nov 2014 19:11:13 -0800
Subject: [PATCH] 5312 libzfs should be decoupled from kernel's zfs_context.h
 Reviewed by: Matthew Ahrens <mahrens@delphix.com> Reviewed by: Will Andrews
 <willa@spectralogic.com> Reviewed by: Andriy Gapon <avg@FreeBSD.org> Approved
 by: Dan McDonald <danmcd@omniti.com>

---
 lib/libzfs/sys/zfs_context.h | 37 +++++++++++++++++++++++++++++
 1 file changed, 37 insertions(+)
 create mode 100644 lib/libzfs/sys/zfs_context.h

diff --git a/lib/libzfs/sys/zfs_context.h b/lib/libzfs/sys/zfs_context.h
new file mode 100644
index 0000000..a2c69f0
--- /dev/null
+++ b/lib/libzfs/sys/zfs_context.h
@@ -0,0 +1,37 @@
+/*
+ * CDDL HEADER START
+ *
+ * This file and its contents are supplied under the terms of the
+ * Common Development and Distribution License ("CDDL"), version 1.0.
+ * You may only use this file in accordance with the terms of version
+ * 1.0 of the CDDL.
+ *
+ * A full copy of the text of the CDDL should have accompanied this
+ * source.  A copy of the CDDL is also available via the Internet at
+ * http://www.illumos.org/license/CDDL.
+ *
+ * CDDL HEADER END
+ */
+/*
+ * Copyright 2014 Spectra Logic Corporation.  All rights reserved.
+ */
+
+#ifndef _SYS_ZFS_CONTEXT_H
+#define	_SYS_ZFS_CONTEXT_H
+
+#ifdef	__cplusplus
+extern "C" {
+#endif
+
+#include <sys/stat.h>
+#include <sys/taskq.h>
+#include <sys/taskq_impl.h>
+#include <sys/debug.h>
+#include <sys/list.h>
+#include <sys/zfs_debug.h>
+
+#ifdef	__cplusplus
+}
+#endif
+
+#endif	/* _SYS_ZFS_CONTEXT_H */
-- 
1.8.3.1

